lib/set-cookie.js

```diff
@@ -58,6 +60,23 @@ function parseString(setCookieValue, options) {
   return cookie;
 }
 
+function parseNameValuePair(nameValuePair) {
+  // Parses name-value-pair according to rfc6265bis draft
+
+  var name = "";
+  var value = "";
+  if (nameValuePair.includes("=")) {
+    var nameValue = nameValuePair.split("=");
```

Comment:
This breaks compatibility with IE11, which *shouldn't* matter, but I've had complaints about it in the past, and since it's easy enough to work around in this case, I'm going to say that we should change it.

```suggestion
  var nameValue = nameValuePair.split("=", 1);
  if (nameValue.length > 1) {
```

(Also, I added a limit of `1` to the split, which I should have done a while ago...)

Reply:
Relatedly, I think the names `nameValuePair` and `nameValue` are a little confusing - I just mixed them up when writing the previous comment. Perhaps we should change it to `nameValueStr` and `nameValueArr`?

Reply:
I don't think a limit of 1 on the split will work. 

`"foo=bar".split("=", 1)` results in `["foo"]`

Also, there is this line (with comment) that makes it seem like there can be multiple `=`s?
```
value = nameValue.join("="); // everything after the first =, joined by a "=" if there was more than one part
```

(I can see tests covering this as well).

A set-cookie-string like this:
```
'foo=bar=bar&foo=foo&John=Doe&Doe=John;'
```

Results in (tested in Chrome):
name: `foo`
value: `bar=bar&foo=foo&John=Doe&Doe=John`

So I think we need to keep the behavior as-is.

I will change the `includes()`, however.

Reply:
🤦 

a) I meant to write `2` instead of `1`
b) you're right that it still wouldn't work, because it truncates.

Sorry about that.